src/main/java/frc/robot/RobotContainer.java

```diff
@@ -39,10 +45,10 @@ public RobotContainer() {
     // c.updateIntakePivotAngle(INTAKE::getIntakePivotAngle_deg);
     c.updateShooterAngle(SHOOTER::getShooterAngle_deg);
     swerveSubsystem = new SwerveSubsystem(new File(Filesystem.getDeployDirectory(), "swerve/maxSwerve"));
-
+    Shuffleboard.getTab("CONFIG").add(new DisabledInstantCommand(() -> {swerveSubsystem.setSwerveOffsets();}));
```

Comment:
We should be more careful about indentation.  Also, we need to look into how to make the name appear on shuffleboard so it doesn't just say "DisabledInstantCommand".

Reply:
Nevermind about the indentation - it was a line-wrap issue in my browser.  Thanks for setting the name!